tests/postgres_tests/test_unaccent.py

```diff
@@ -42,6 +43,24 @@ def test_unaccent_chained(self):
             ordered=False
         )
 
+    def test_unaccent_chained_with_conforming_strings_off(self):
+        """
+        Chained unaccent causes the comparison string to be escaped.
+        Make sure the escaping SQL is valid even if the DB has standard_conforming_strings OFF.
+        """
+
+        with connection.cursor() as cur:
+            cur.execute("SET standard_conforming_strings TO OFF;")
+            try:
+                self.assertQuerysetEqual(
+                    self.Model.objects.filter(field__unaccent__endswith="éÖ"),
+                    ["àéÖ", "aeO"],
+                    transform=lambda instance: instance.field,
+                    ordered=False)
+            except DatabaseError:
+                self.fail("DatabaseError should not have occurred.")
```

Comment:
Remove try/except as that hides the original exception and makes debugging more difficult (see c9ae09addffb839403312131d4251e9d8b454508).